From 9a0962478463154d3da1e81abdc99c6904e5a15f Mon Sep 17 00:00:00 2001
From: Andy Ross <andrew.j.ross@intel.com>
Date: Thu, 17 Jul 2014 08:54:49 -0700
Subject: [PATCH 080/117] atomisp_gmin_platform: Update ECS config variables

BIOS fixed the _HID, but we still had the original here

Signed-off-by: Andy Ross <andrew.j.ross@intel.com>
---
 arch/x86/platform/intel-mid/atomisp_gmin_platform.c | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
index 08fa4af..5ad0f72 100644
--- a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
+++ b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
@@ -240,8 +240,14 @@ static const struct gmin_cfg_var mrd7_vars[] = {
 	{ "CRMT1040:00_CsiPort",  "0" },
 	{ "CRMT1040:00_CsiLanes", "1" },
 	{ "CRMT1040:00_CsiFmt",   "13" },
-	{ "CRMT1040:00_CsiBayer", "0" }, /* FIXME: correct? */
+	{ "CRMT1040:00_CsiBayer", "0" },
 	{ "CRMT1040:00_CamClk", "1" },
+	{ "INT33F0:00_CamType",  "1" },
+	{ "INT33F0:00_CsiPort",  "0" },
+	{ "INT33F0:00_CsiLanes", "1" },
+	{ "INT33F0:00_CsiFmt",   "13" },
+	{ "INT33F0:00_CsiBayer", "0" },
+	{ "INT33F0:00_CamClk", "1" },
 	{},
 };
 
-- 
1.9.3

